ext/RMagick/rmimage.c

```diff
@@ -2557,11 +2558,13 @@ Image_color_histogram(VALUE self)
     Image *image, *dc_copy = NULL;
     VALUE hash, pixel;
     size_t x, colors;
-    ColorPacket *histogram;
     ExceptionInfo *exception;
+    ColorPacket *histogram;
 
     image = rm_check_destroyed(self);
 
+    exception = AcquireExceptionInfo();
```

Comment:
I think this line does not need to move.
It is easier to understand when it is executed immediately before use.

Reply:
I moved it a bit earlier because this needs to be done in ImageMagick 7. The diff with ImageMagick 7 will be cleaner if this is done first.